src/mainWorker.cpp

```diff
@@ -7,6 +7,13 @@
 
 #include "ui.hpp"
 
+/* TODO move to a class during refactor */
+const uint8_t VMAJOR = 1;
+const uint8_t VMINOR = 2;
+const uint8_t VREVISION = 5;
+const char VTAG = 'd';
+const mab::version_ut mdtoolVersion = {{VTAG, VREVISION, VMINOR, VMAJOR}};
```

Comment:
consider
https://www.youtube.com/watch?v=rQhBECyA6ew&ab_channel=TheCherno
(may be not applicable at this very case (small codebase) but worth to remember)